internal/pkg/state/fixtures.go

```diff
@@ -50,6 +60,16 @@ func NewProjectSnapshot(s *State) (*fixtures.ProjectSnapshot, error) {
 		c.Rows = append(c.Rows, r)
 	}
 
+	schedules, err := testProject.SchedulerApi().ListSchedules()
+	if err != nil {
+		panic(err)
```

Comment:
Tu by som uprednostnil `return nil, err` ... ked bude mat API nejaky vypadok, tak to aspon nezastavi vsetky ostatne testy, a bude sa jednoduchsie hladat pricina.

O uroven vyssie je potom:
![image](https://user-images.githubusercontent.com/19371734/140077598-bce4c876-f76b-47c2-b515-ba5c8108cd15.png)

Reply:
jasně 👍